targets/XPlatCppSdk/source/build/Windows/TestApp.vcxproj.ejs

```diff
@@ -63,7 +63,7 @@
       <SDLCheck>true</SDLCheck>
       <PreprocessorDefinitions>_DEBUG;_CONSOLE;WIN32;<%- extraDefines %>%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <ConformanceMode>true</ConformanceMode>
-      <AdditionalIncludeDirectories>$(TestSourceDir)\TestApp;$(SdkSourceDir)\include;$(ExternalDir)\deps-$(Configuration)\include</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(TestSourceDir)\TestApp;$(SdkSourceDir)\include;$(ExternalDir)\jsoncpp\include</AdditionalIncludeDirectories>
```

Comment:
Random place to put the comment : 
Should we consider adding another test project using nuget for windows and xbox? It will be the same code but a new vcxproj file. This will help us validate the nugets that we push.